lib/manager/gitlabci-include/extract.ts

```diff
@@ -26,22 +25,35 @@ function extractDepFromInclude(includeObj: {
   return dep;
 }
 
-export function extractPackageFile(
+async function extractDepsFromIncludeLocal(includeObj: {
+  local: string;
+}): Promise<PackageDependency[] | null> {
+  const content = await readLocalFile(includeObj.local, 'utf8');
+  const deps = gitlabci.extractPackageFile(content)?.deps;
+  return deps;
+}
+
+export async function extractPackageFile(
   content: string,
   _packageFile: string,
   config: ExtractConfig
-): PackageFile | null {
+): Promise<PackageFile | null> {
   const deps: PackageDependency[] = [];
   try {
     const doc = yaml.safeLoad(content, { json: true });
     if (doc?.include && is.array(doc.include)) {
       for (const includeObj of doc.include) {
-        const dep = extractDepFromInclude(includeObj);
-        if (dep) {
+        if (includeObj.file && includeObj.project) {
+          const dep = extractDepFromIncludeFile(includeObj);
           if (config.endpoint) {
             dep.registryUrls = [config.endpoint.replace(/\/api\/v4\/?/, '')];
           }
           deps.push(dep);
+        } else if (includeObj.local) {
+          const includedDeps = await extractDepsFromIncludeLocal(includeObj);
+          for (const includedDep of includedDeps) {
```

Comment:
`includedDeps` needs to be checked because it can be `null` which isn't iterable.

Reply:
added includedDeps null check